src/testdir/Make_ming.mak

```diff
@@ -22,7 +22,7 @@ default: nongui
 include Make_all.mak
 
 # Explicit dependencies.
-test_options.res test_alot.res: opt_test.vim
```

Comment:
you dropped one dependency here: `test_alot.res`. Is that intentionally?

Reply:
*test_options.vim* is no longer sourced by *test_alot.vim* at bb962262.